core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/RegionExecutionCoordinator.scala

```diff
@@ -62,7 +62,11 @@ class RegionExecutionCoordinator(
     })
 
     // update UI
-    asyncRPCClient.sendToClient(ExecutionStatsUpdate(regionExecution.getStats))
+    asyncRPCClient.sendToClient(
+      ExecutionStatsUpdate(
+        workflowExecution.getAllRegionExecutionsStats
```

Comment:
I think we only need to update the current region rather than the entire workflow.

Reply:
https://github.com/Texera/texera/pull/2570#discussion_r1554849374